### PR TITLE
Docs: Add information about class mapper

### PR DESCRIPTION
**Pull request by katakumby**
I think it would be great to include information about setClassNameMapper property into docs as well

### CODE REVIEW

**Comment by oojacoboo** [comment]
@katakumby Thanks for the PR.  I think under most circumstances this isn't a necessity at all.  Your composer autoload should take care of this.  Do you have a specific use case here?  Or, is this more about documenting the public API?  I'm not entirely opposed to this, but it does not need to be made so prominent in the config docs, and should be prefaced with information about Composer, the preferred way of handling the mapping.  We don't really want people abusing this.

**Comment by katakumby** [comment]
This is more about documenting public API and you are right it don't have to be so prominent, in my case i needed to register custom directory because of customized non framework environment, when using that method i was able to properly integrate https://github.com/ThingEngineer/PHP-MySQLi-Database-Class existing models to support graphqlite in separated environment's with shared models, im doing it that way to have as less as possible dependencies across environments/apps.

File: website/versioned_docs/version-4.1/other_frameworks.md [hunk 1]
```diff
@@ -54,6 +54,8 @@ The `SchemaFactory` class also comes with a number of methods that you can use t
 $factory->setAuthenticationService(new VoidAuthenticationService());
 // Configure an authorization service (to resolve the @Right annotations).
 $factory->setAuthorizationService(new VoidAuthorizationService());
+//Add class mapper to map a class names to one or many possible file names according to PSR-0 or PSR-4
```
Comment (oojacoboo) [inline website/versioned_docs/version-4.1/other_frameworks.md:57]: Please follow the comment code-style, space after `//`

File: website/versioned_docs/version-4.1/other_frameworks.md [hunk 2]
```diff
@@ -54,6 +54,8 @@ The `SchemaFactory` class also comes with a number of methods that you can use t
...
+$factory->setClassNameMapper(new ClassNameMapper());
```
Comment (oojacoboo) [inline website/versioned_docs/version-4.1/other_frameworks.md:58]: Please move this further down the calls, ie. priority.

**Review by oojacoboo — CHANGES_REQUESTED** [review]

**Comment by oojacoboo** [comment]
The docs have been updated quite a bit.  Please review for merging.

**Pull request closed without merging**